frontend/integration-tests/views/secrets.view.ts

```diff
@@ -77,3 +97,35 @@ export const editSecret = async(ns: string, name: string, updateForm: Function)
   await saveButton.click();
   expect(crudView.errorMessage.isPresent()).toBe(false);
 };
+
+export const addSecretToWorkloadAsEnv = async(workloadname: string, EnvPrefix: string) => {
+  await clickAddSecretToWorkload();
+  await browser.wait(until.presenceOf(selectWorkloadBtn));
+  await selectWorkloadBtn.click();
+  await browser.wait(until.presenceOf($('li[role=option]')), 5000);
+  await element(by.cssContainingText('li[role=option] a', workloadname)).click();
+  await addSecretAsEnv.click();
+  await addSecretAsEnvPrefixInput.sendKeys(EnvPrefix);
+  await monitoringView.modalConfirmButton.click();
```

Comment:
```suggestion
  await $('#confirm-action').click();
```

nit: would rather use the selector here, then importing the whole `monitoring.view.ts`

Reply:
Ok, I will update